components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationMgtUtil.java

```diff
@@ -1007,6 +1007,10 @@ public static void startTenantFlow(String tenantDomain) throws IdentityApplicati
 
```

Comment:
Make sure this is called from all the application service impl.

Reply:
Fixed with [802eef3](https://github.com/wso2/carbon-identity-framework/pull/4361/commits/802eef3e68825657fbf77b4e1ae3fe726ed4bebc)